wiki/Rules/th.md

```diff
@@ -0,0 +1,66 @@
+8. **อย่าโฆษณาโปรแกรมพูดคุยภายนอก รวมถึง เซิฟเวอร์/กลุ่ม Discord, Teamspeak หรือ Skype** แชท osu! นั้นก็มีไว้สำหรับ osu! โดยเฉพาะ มันไม่ได้มีไว้สำหรับการส่งลิงก์เข้าเซิฟเวอร์ส่วนตัวของคุณ ส่งมันในข้อความส่วนตัว (PM) ถ้ามันจำเป็นจริง ๆ เท่านั้น
+9. **ในช่องแชทที่มีภาษาระบุ คุณต้องพูดภาษานั้น (เช่น พูดอังกฤษใน `#english`, พูดไทยใน `#thai`) ถ้าคุณกำลังมีส่วนร่วมในการสนทนา** ในช่องที่ไม่ได้มีภาษาบอกไว้ มันคือช่องที่ต้องพูดอังกฤษเท่านั้น แต่ในช่อง `#lobby` และ `#help` นั้นเป็นช่องที่พูดได้ทุกภาษาไม่เหมือนช่องอื่น ๆ *ทีมงานของ [ผู้ดูแล](/wiki/People/The_Team/Global_Moderation_Team) ไม่จำเป็นจะต้องทำตามกฏข้อนี้ระหว่างปฏิบัติหน้าที่*
+
+## กฏของฟอรั่ม
+
+1. **โพสต์เฉพาะสิ่งที่มีประโยชน์, น่าสนใจ, หรือหัวข้อที่ดึงดูดความสนใจ** ฟอรั่มนั้นมีไว้สำหรับการสนทนากัน ดังนั้นอย่าทำอะไรชุ่ย ๆ ที่ทำให้เสียเวลาคนอื่น
+2. **ทำให้โพสต์และการตอบกลับโพสต์ของคุณเหมาะสมกับทุกเพศทุกวัย** เรื่องพวก NSFW นั้นไม่เหมาะกับชุมชนที่มีหลากหลายวัย แต่ว่ามันก็ไม่มีอะไรจะหยุดคุณในการคุยเรื่องพวกนี้ในแชทส่วนตัวนะ
+3. **ให้คำติชมที่สมเหตุสมผล โดยที่ไม่ใช้อารมณ์ส่วนตัวเข้ามาเกี่ยว** การติชมเป็นสิ่งที่สุดยอดไปเลยถ้าทำได้อย่างถูกต้อง แต่ถ้าคุณจะต้องการโจมตีเป็นการส่วนตัวเพื่อที่จะแสดงว่าตัวเองถูก คุณควรรู้ไว้ว่ามันเป็นการติชมที่ผิด และคุณควรรู้สึกแย่
+4. **อย่าดันกระทู้ (threads) ที่เก่าแล้วเข้ามา ถ้าการสนทนามันเงียบไปนานแล้ว** ถ้าหัวข้อไหนนั้นมันเงียบไปเป็นปีแล้ว มันก็คงจะไม่เข้ายุคแล้วหละ ดังนั้นปล่อยให้มันหายไป โพสต์อันใหม่, หรือหาว่ามีใครตอบคำถามคุณหรือยัง
+5. **ทำให้ลายเซ็น และ รูปโปรไฟล์ของคุณสมเหตุสมผล** ถ้ามันเคลื่อนไหวได้ มันไม่ควรจะกระพริบมากจนเกินไป และภาพก็ควรจะเล็กพอที่จะทำให้คนที่มีการเชื่อมต่อที่ช้าเข้าถึงได้ ที่ดีที่สุดคือต่ำกว่า 500 กิโลไบต์ (500 KB) ถ้าเป็นไปได้
+6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** ของพวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลด หรือเล่นเนื้อหา โดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย
+7. **เราไม่อนุญาติ การลิงก์ไปแอพพลิเคชั่นหรือเครื่องมืออื่น ๆ ทีมีจุดมุ่งหมายไม่ชัดเจน** เอาง่าย ๆ เลย ถ้าคิดว่าเครื่องมือที่กล่าวมาผิดกฏหมายหรือไม่ เราไม่อนุญาติให้โพสต์สิ่งเหล่านั้น
+
+## กฏการอัปโหลดบีทแมพ
+
+*โน้ต: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่ แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)
+
+กฏพวกนี้มีผลต่อบีทแมพที่ถูกอัปโหลดผ่าน [Beatmap Submission System](/wiki/Glossary#bss) ภายในเกม กระทู้/การสนทนา จะถูกสร้างขึ้นด้วย osu! client ในระหว่างที่บีทแมพถูกอัปโหลดผ่าน [เครื่องมือแก้ไขบีทแมพ](/wiki/Beatmap_Editor)
+
+1. **เช็คให้แน่ใจว่าคุณมีสิทธิในการใช้อะไรก็ตามที่อยู่ในบีทแมพของคุณ** นี่รวมถึง เพลง, วีดีโอ, hitsounds, ภาพต่าง ๆ , หรือเนื้อหาอะไรก็ตามที่ไม่ใช่ของคุณเอง ถ้าคุณไม่แน่ใจว่าหาเนื้อหาที่ไหนที่ไม่ต้องเสียตังค์ในการใช้ ลองไปเช็คที่คลัง [Featured Artist](https://osu.ppy.sh/beatmaps/artists) ของเราสิ ที่นั้นมีเพลงที่จดสิขสิทธิ์แล้วสำหรับการใช้ใน osu! 100% และเรามี [หลักเกณฑ์การใช้งานเนื้อหา](Content_Usage_Guidelines)
```

Comment:
```suggestion
1. **เช็คให้แน่ใจว่าคุณมีสิทธิในการใช้อะไรก็ตามที่อยู่ในบีทแมพของคุณ** นี่รวมถึง เพลง, วีดีโอ, hitsounds, ภาพต่าง ๆ , หรือเนื้อหาอะไรก็ตามที่ไม่ใช่ของคุณเอง ถ้าคุณไม่แน่ใจว่าหาเนื้อหาที่ไหนที่ไม่ต้องเสียตังค์ในการใช้ ลองไปเช็คที่คลัง [Featured Artist](https://osu.ppy.sh/beatmaps/artists) ของเราสิ ที่นั้นมีเพลงที่จดสิขสิทธิ์แล้วสำหรับการใช้ใน osu! 100% และดูข้อมูลเพิ่มเติมได้ที่ [หลักเกณฑ์การใช้งานเนื้อหา](Content_Usage_Guidelines)
```

Reply:
Changed to "คุณสามารถดูข้อมูลเพิ่มเติมได้ที่"